docs/hello-neighbour.yaml

```diff
@@ -275,14 +255,16 @@ paths:
           description: "Question updated"
           schema:
             $ref: "#/definitions/Question"
-        404:
-          description: "Question not found"
-          schema:
-            $ref: "#/definitions/ErrorResponse"
```

Comment:
I don't think this should be deleted. You may be trying to update a question which doesn't actually exist at all.